processing-service-core/api/swagger/swagger.yaml

```diff
@@ -38,6 +38,8 @@ tags:
     description: Operations for Conditions on Processing Rules.
   - name: Workflows
     description: Workflow operations.
+  - name: Global Config
+    description: Default Config for Tenant.
```

Comment:
Instead of "Default Config for Tenant" I think this should just be "Global Configurations" I don't think they should mention a tenant.